charts/cluster-autoscaler/templates/vpa.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.vpa.enabled -}}
+apiVersion: "autoscaling.k8s.io/{{ .Values.vpa.apiVersion }}"
+kind: VerticalPodAutoscaler
+metadata:
+  labels:
+{{ include "cluster-autoscaler.labels" . | indent 4 }}
+  name: {{ template "cluster-autoscaler.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  targetRef:
+    apiVersion: {{ template "deployment.apiVersion" . }}
+    kind: Deployment
+    name: {{ template "cluster-autoscaler.fullname" . }}
+  updatePolicy:
+    updateMode: "Auto"
```

Comment:
I see people run their VPA in `updateMode: off` for a while to observe if anything weird would happen when they switched on `updateMode: auto` – so maybe this is an option you'd want to make configurable?

Reply:
I didn't think about that. I'll add it